ci/install-extra-builddeps.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+# Install build dependencies not in the cosa buildroot already
+set -xeuo pipefail
+if ! command -v cxxbridge; then
+    # Keep this in sync with Cargo.lock
+    cargo install --root=/usr cxxbridge-cmd --version 1.0.18
```

Comment:
I think this version string can be directly extracted from the lockfile via grep/jq, so that it cannot go out-of-sync.

Reply:
Ah right, this made me remember about `cargo metadata --format-version 1`, added a commit which uses that and unpinned it so cxx will be managed via dependabot too!

Reply:
I didn't know about that, even nicer!